web/packages/teleport/src/Console/DocumentSsh/DocumentSsh.test.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { render, screen } from 'design/utils/testing';
+import { MemoryRouter } from 'react-router';
+import 'jest-canvas-mock';
+
+import { allAccessAcl } from 'teleport/mocks/contexts';
+
+import ConsoleContext from '../consoleContext';
+import ConsoleContextProvider from '../consoleContextProvider';
+
+import DocumentSsh from '.';
+
+test('file transfer buttons are disabled if user does not have access', async () => {
```

Comment:
this component didn't have any tests at all previously. I've only added tests relevant to this PR